SavedInstances/SavedInstances.lua

```diff
@@ -48,6 +49,36 @@ vars.Indicators = {
   BLANK = "None",
 }
 
+local KeystoneAbbrev = {
```

Comment:
My mind thinks in lua and English, so I didn't think about localizing the abbreviations. Definitely agree though.